ci/report-tests.ps1

```diff
@@ -42,18 +42,67 @@ if (Test-Path "$test_result_dir\index.html" -PathType Leaf) {
         --style info
 }
 
-## Read the test results, and pass/fail the build accordingly 
+# Read the test results
 $results_path = Join-Path -Path $test_result_dir -ChildPath "index.json"
 $results_json = Get-Content $results_path -Raw
+$test_results_obj = ConvertFrom-Json $results_json
+$tests_passed = $test_results_obj.failed -eq 0
 
-$results_obj = ConvertFrom-Json $results_json
+# Upload artifacts to Buildkite, merge all output streams to extract artifact ID in the Slack message generation
+$ErrorActionPreference = "Continue" # For some reason every piece of output being piped is considered an error
```

Comment:
Hmmm, is this a json parsing error perhaps?

Reply:
Line 52 (`$upload_output = buildkite-agent artifact upload "$test_result_dir\*" *>&1 | %{ "$_" } | Out-String`) is the one that breaks. This script doesn't attempt to parse the output, it just throws an error 😢